core/eolearn/tests/test_eodata_io.py

```diff
@@ -269,8 +288,12 @@ def test_cleanup_different_compression(fs_loader, eopatch):
     with fs_loader() as temp_fs:
         temp_fs.makedir(folder)
 
-        save_compressed_task = SaveTask(folder, filesystem=temp_fs, compress_level=9, overwrite_permission=1)
-        save_noncompressed_task = SaveTask(folder, filesystem=temp_fs, compress_level=0, overwrite_permission=1)
+        save_compressed_task = SaveTask(
+            folder, filesystem=temp_fs, compress_level=9, overwrite_permission="OVERWRITE_FEATURES"
+        )
+        save_noncompressed_task = SaveTask(
+            folder, filesystem=temp_fs, compress_level=0, overwrite_permission="OVERWRITE_FEATURES"
```

Comment:
Should there also be some usage of the lowercase permissions?

Reply:
not sure, i consider that a favor to users since it's not explicitly mentioned anywhere. Intended use is `OverwritePermission.ADD_ONLY`.

we could if you consider it relevant enough